persistence/src/main/resources/db/migration/schema/V1_15__AlterStateHistoryTable.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE states_history ADD COLUMN position_id BIGINT;
```

Comment:
Be careful with the naming when you merge master to this branch, because others create sql files too with version 15.